### PR TITLE
Laboratory work #1, Maksimova Anna - 23FPL2

### PR DESCRIPTION
**Pull request by Piapina**

### CODE REVIEW

File: lab_1_classify_by_unigrams/main.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 Language detection
 """
+
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:6]: revert

File: lab_1_classify_by_unigrams/main.py [hunk 2]
```diff
@@ -20,6 +22,12 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned
     """
+    text = text.lower()
+    lst = []
+    for letter in text:
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:27]: Too hard. Can you do this work with one loop?

File: lab_1_classify_by_unigrams/main.py [hunk 3]
```diff
@@ -67,8 +75,8 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
 
 def compare_profiles(
-    unknown_profile: dict[str, str | dict[str, float]],
-    profile_to_compare: dict[str, str | dict[str, float]],
+        unknown_profile: dict[str, str | dict[str, float]],
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:78]: revert

File: lab_1_classify_by_unigrams/main.py [hunk 4]
```diff
@@ -87,9 +95,9 @@ def compare_profiles(
 
 
 def detect_language(
-    unknown_profile: dict[str, str | dict[str, float]],
-    profile_1: dict[str, str | dict[str, float]],
-    profile_2: dict[str, str | dict[str, float]],
+        unknown_profile: dict[str, str | dict[str, float]],
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:98]: revert

File: lab_1_classify_by_unigrams/main.py [hunk 5]
```diff
@@ -152,7 +160,7 @@ def collect_profiles(paths_to_profiles: list) -> list[dict[str, str | dict[str,
 
 
 def detect_language_advanced(
-    unknown_profile: dict[str, str | dict[str, float]], known_profiles: list
+        unknown_profile: dict[str, str | dict[str, float]], known_profiles: list
```
Comment (artyomtugaryov) [inline lab_1_classify_by_unigrams/main.py:163]: revert

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_1_classify_by_unigrams/main.py (hunk 2 above)
Comment (Piapina) [reply lab_1_classify_by_unigrams/main.py:28]: shortened

File: lab_1_classify_by_unigrams/main.py (hunk 1 above)
Comment (Piapina) [reply lab_1_classify_by_unigrams/main.py:6]: +

File: lab_1_classify_by_unigrams/main.py (hunk 3 above)
Comment (Piapina) [reply lab_1_classify_by_unigrams/main.py]: +

File: lab_1_classify_by_unigrams/main.py (hunk 4 above)
Comment (Piapina) [reply lab_1_classify_by_unigrams/main.py:113]: +

File: lab_1_classify_by_unigrams/main.py (hunk 5 above)
Comment (Piapina) [reply lab_1_classify_by_unigrams/main.py:178]: +

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
What about 6?

**Pull request closed without merging**